src/styles/stars.scss

```diff
@@ -0,0 +1,19 @@
+.stars {
+  display: flex;
+  flex-direction: row;
+  justify-content: space-evenly;
+  align-items: baseline;
+  width: 96px;
+  height: 16px;
+
+  &__star {
+    background-image: url(/src/images/star.svg);
+    background-repeat: no-repeat;
+    height: 16px;
+    width: 16px;
```

Comment:
The paths to the images should be relative to the CSS file. If the `src` directory is not the root directory for your server, the path should be adjusted accordingly. For instance, if your CSS files are served from a `styles` directory, the path should be `../images/star.svg` and `../images/star-active.svg`.